app/models/filter.rb

```diff
@@ -16,6 +16,7 @@ class Filter < ApplicationRecord
 
   belongs_to :author, class_name: 'User'
   belongs_to :tenant
+  has_many :filter_subscriptions, dependent: :destroy
```

Comment:
Na urovni db to nechceme radsej vyriesit (cascade) ci robievame to takto?

Reply:
Nieco mame tak, nieco tak, ale castejsie to riesime az v modeli

Reply:
Asi to je safe, nakoniec je tam FK cize keby to nemalo nieco zmazat tak to padne na db errore.